src/app/services.service.ts

```diff
@@ -1,17 +1,30 @@
 import { Injectable } from '@angular/core';
-
+import { HttpClient, HttpHeaders } from '@angular/common/http';
+import { Observable } from 'rxjs';
 @Injectable({
   providedIn: 'root'
 })
 export class ServicesService {
   data:any
-  constructor() { }
+  index:any
+  constructor(private http:HttpClient) { }
+  setIndex(data:any){
+    this.index = data
+  }
 
+  getIndex(){
+    return this.index
+  }
+  
   setItem(value:any){
     this.data = value
     console.log(this.data)
   }
   getItem(){
     return this.data
   }
+  getData():Observable<any>{
```

Comment:
make a constant file and add base url of the api in that and the parameters if static should be fetched from constants file too